packages/core/src/awsService/appBuilder/models/metadata.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Let's use a clearer name for this. At least something like `serverlessland-metadata.json` (or maybe only `metadata.json`, but inside a `serverlessland/` directory or something)

Reply:
Changed it with `serverlessland-metadata.json`